Explorer/Assets/DCL/Multiplayer/Chats/DebugPanelChatMessageBus.cs

```diff
@@ -0,0 +1,41 @@
+using DCL.Chat;
+using DCL.DebugUtilities;
+using System;
+using Utility;
+
+namespace DCL.Multiplayer.Chats
+{
+    public class DebugPanelChatMessageBus : IChatMessagesBus
+    {
+        private readonly IChatMessagesBus origin;
+
+        public event Action<ChatMessage>? OnMessageAdded;
+
+        public DebugPanelChatMessageBus(IChatMessagesBus origin, IDebugContainerBuilder debugBuilder)
+        {
+            this.origin = origin;
+            this.origin.OnMessageAdded += OriginOnOnMessageAdded;
```

Comment:
I would suggest to follow our code style rule here, since double OnOn looks strange 
`origing.MessageAdded += OriginOnMessageAdded;`